KsApi/Service.swift

```diff
@@ -19,21 +19,24 @@ public struct Service: ServiceType {
   public let language: String
   public let currency: String
   public let buildVersion: String
+  public let deviceIdentifier: String
```

Comment:
In order to not make tons of changes I haven't alphabetized these properties for now. I have plans to do this when we do some refactoring of this type soon.